### PR TITLE
Update the user group for Cluster API release team

### PR DESCRIPTION
**Pull request by joekr**
This PR updates the slack user group for the Cluster API release team.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Review by ykakarap — APPROVED** [review]
/lgtm

**Review by furkatgofurov7 — COMMENTED** [review]
Thanks, lgtm except a small nit:

The list seemed to be sorted alphabetically previously, maybe we should keep the new one the same way?

File: communication/slack-config/usergroups.yaml [hunk 1]
```diff
@@ -249,20 +249,21 @@ usergroups:
...
       - aniruddha2000
       - CecileRobertMichon
-      - cpanato
-      - dntosas
+      - chiukapoor
+      - chrischdi
       - furkatgofurov7
+      - g-gaston
       - hackeramitkumar
-      - joekr
```
Comment (furkatgofurov7) [inline communication/slack-config/usergroups.yaml:259]: hmm, so you are being removed from the team here :)

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Review by chrischdi — COMMENTED** [review]
/lgtm

**Pull request merged**